crates/cheatcodes/src/evm.rs

```diff
@@ -224,6 +227,28 @@ impl Cheatcode for resumeGasMeteringCall {
     }
 }
 
+impl Cheatcode for lastGasUsedCall {
+    fn apply(&self, state: &mut Cheatcodes) -> Result {
+        let Self {} = self;
+        ensure!(
+            state.config.isolate,
+            "`lastGasUsed` is only available in isolated mode (`--isolate`)"
+        );
```

Comment:
I've opted to throw an error on non-isolation mode as the cheatcode has no use in that scenario